tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -281,12 +281,12 @@ public void testInvalidTomlFile() throws BallerinaTestException {
         return new Object[][]{
                 {"invalidComplexArray", "[Config.toml:(2:17,2:41)] configurable variable 'main:intComplexArr' with " +
                         "type '(int[] & readonly)[] & readonly' is not supported"},
-                {"invalidRecordField", "[Config.toml:(4:1,4:40)] field type '(string[][] & readonly)' in configurable" +
```

Comment:
I created an issue to change the message https://github.com/ballerina-platform/ballerina-lang/issues/29688
let's address that in beta.